### ISSUE TITLE
styleguide applied to  overviewpages

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#44]
## User story
1. As a {HR-employee}
2. I want / need {a beautiful UI}
3. So that {I get very excited and don't fall asleep while working, and in order to work efficiently}

## Acceptance criteria
* styleguide rules applied to all pages, amongst them
* mockup is taken as reference

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
